atd-toolbox/data_model/python/populate_edits_sql/people.sql

```diff
@@ -0,0 +1,193 @@
+set client_min_messages to warning;
+
+-- remove all names from old people tables for non-fatalities
+-- this can take more than a minute per table
+update atd_txdot_person set prsn_last_name = null, prsn_first_name = null, prsn_mid_name = null
```

Comment:
is this because in the new data model we are only keeping names for fatalities?

Reply:
yep!